.github/workflows/stale.yaml

```diff
@@ -0,0 +1,25 @@
+name: Close Stale Pull Requests
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Run every day at midnight
+
+jobs:
+  close_stale_prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close stale pull requests
+        uses: actions/stale@v9.1.0
+        with:
+          days-before-pr-stale: 5
+          days-before-pr-close: 5
+          stale-pr-label: stale
+          stale-pr-message: |
+            This pull request has been marked as stale because it has been
+            inactive for more than 5 days. Please update this pull request
+            or it will be automatically closed in 5 days.
+            
```

Comment:
missing empty new line at the end of the file. maybe line 25 is not empty ... maybe if you just remove the spaces in the line it should work.

you can change the days-before-pr-close to 30 .. just to ensure to not close a PR that shouldn't be closed.

Reply:
also update the stale-pr-message accordingly pls.